robot_skills/src/robot_skills/util/entity.py

```diff
@@ -239,11 +241,9 @@ def from_entity_info(e):
     if e.type == 'person':
         try:
             pp_dict = yaml.load(e.data)
-            tags_dict = dict(zip(pp_dict['tagnames'], pp_dict['tags']))
             del pp_dict['position']
```

Comment:
Why the deletion of data from the dict? Do we need to sync the person properties to (new) message definitions?

Reply:
Position is a property of the Entity, not of Entity.PersonProperties